src/main/java/com/github/vkorobkov/jfixtures/processor/ColumnProcessor.java

```diff
@@ -39,7 +41,7 @@ private FixtureValue referredColumn(String table, String referredTable, FixtureV
             throw new ProcessorException(message);
         }*/
 
-        return referredRow.getValues().get(Processor.PK_COLUMN_NAME);
+        return referredRow.getValues().get(getTablesConfig().getCustomColumnForPk(referredTable));
```

Comment:
The comment above makes sense now and this code could be uncommented and tested since we can turn of auto generation of PK